src/components/textarea/textarea.jsx

```diff
@@ -0,0 +1,129 @@
+import PropTypes from 'prop-types';
+import React, {
+  forwardRef,
+  useEffect,
+  useRef,
+  useImperativeHandle,
+} from 'react';
+import cautionSvg from '../../svgs/caution.svg';
+import FormControl from '../form-control/form-control.jsx';
+import Icon from '../icon/icon.jsx';
+import useMounted from '../../hooks/use-mounted.js';
+import useValidation from '../../hooks/use-validation.jsx';
+import styles from './textarea.css';
+
+function getClassName(className, validationMessage) {
+  if (className) return className;
+  return validationMessage ? styles['invalid-textarea'] : styles.textarea;
+}
+
+const Textarea = forwardRef(function Textarea({
+  className,
+  cssContainer,
+  id,
+  label,
+  name,
+  onChange,
+  onBlur,
+  placeholder,
+  readOnly,
+  required,
+  validationMessage,
+  value,
+}, forwardedRef) {
+  const fallbackRef = useRef();
+  const ref = forwardedRef || fallbackRef;
+  const textareaRef = useRef();
+  const mounted = useMounted();
+  const [validationMessageValue, validate] = useValidation(validationMessage, textareaRef);
+
+  function blurHandler(event) {
+    validate();
+    onBlur(event);
+  }
+
+  function changeHandler(event) {
+    validate();
```

Comment:
I think we will _not_ want this. Can you check this with Anne when ya pair with her? The validation spec is:

- On blur, the input goes from grey to red or red to grey
- On blur, the native validation message appears or disappears
  - Server validation messages (provided by validationMessage API) are always shown since we assume some other layer is managing it closely